src/js/components/header/mobile.js

```diff
@@ -9,20 +9,26 @@ function toggleClass (element, className) {
   }
 }
 
-function mobileInit () {
-  var navElements = select('.usa-menu-btn, .usa-overlay, .usa-nav-close');
+function handleNavElements (e) {
+
   var toggleElements = select('.usa-overlay, .usa-nav');
   var navCloseElement = select('.usa-nav-close')[ 0 ];
```

Comment:
@shawnbot took a bit of time to take a closer look at this approach from holistic stand point. I agree with @rogeruiz on the multiple navs and also that we probably don't want to scope the code base to encourage that type of behavior. We also don't focus the scope in this way throughout the current JS code base and think it makes sense for us to be consistent. 

However, I did make the `clickHandler` available globally in 9754434. 😄